index.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const Base = require('bfx-facs-base')
+const axios = require('axios')
```

Comment:
using `.default` gives better intellisense and usage although they're both the same functionality wise:
```javascript
const axios = require('axios').default
```

```javascript
// file: bfx-facs-places-google/node_modules/axios/lib/axios.js
module.exports = axios;

// Allow use of default import syntax in TypeScript
module.exports.default = axios;
```